ide/che-core-ide-templates/src/main/resources/samples.json

```diff
@@ -1,5 +1,66 @@
 [
   {
+    "name": "web-java-petclinic",
+    "displayName": "web-java-petclinic",
+    "path": "/web-java-petclinic",
+    "description": "Petclinic app using MySQL as persistent storage",
+    "projectType": "maven",
+    "mixins": [],
+    "attributes": {
+      "language": [
+        "java"
+      ]
+    },
+    "modules": [],
+    "problems": [],
+    "source": {
+      "type": "git",
+      "location": "https://github.com/che-samples/web-java-spring-petclinic.git",
+      "parameters": {
+        "branch":"mysql"
+      }
+    },
+    "commands": [
+      {
+        "name": "build",
+        "type": "mvn",
+        "commandLine": "mvn -f ${current.project.path} clean install -DskipTests",
+        "attributes": {
```

Comment:
Can you check this macro with M1 - according to what Vlad produced, this macro is now in a different format.  Should we update it?

https://eclipse-che.readme.io/docs/commands